src/index.js

```diff
@@ -33,8 +26,39 @@ items.forEach((item) => {
 
   input.addEventListener('change', () => {
     updateCompleted(item, input);
-    saveLocalStorage(items);
+    saveLocalStorage(itemsArray);
   });
 
   todos.appendChild(div);
+  const vertical = div.querySelector('.vertical');
+  const trash = div.querySelector('.trash');
+  const edit = div.querySelector('.edit');
+  edit.addEventListener('change', () => {
+    editTodo(itemsArray, div.id, edit.value);
+    saveLocalStorage(itemsArray);
+  });
+  vertical.addEventListener('click', () => {
+    vertical.style.display = 'none';
+    trash.style.display = 'block';
+  });
+  trash.addEventListener('click', () => {
+    div.remove();
+    itemsArray = itemsArray.filter((i) => removeTodos(i, div.id));
+    saveLocalStorage(itemsArray);
+  });
+});
+
+const form = document.getElementById('form-id');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const { value } = form.querySelector('input');
+  addTodo(itemsArray, value);
+  saveLocalStorage(itemsArray);
+  window.location.reload();
+});
+
+const clearButton = document.querySelector('#button');
+clearButton.addEventListener('click', () => {
```

Comment:
- [ ] I tried adding a few tasks manually and I was not able to use the "Clear all completed" button/functionality. Please, kindly fix this issue :smiley: :raised_hands: